src/WaitCommand.php

```diff
@@ -42,13 +41,21 @@ public function wait($files, Env $env, PlatformShFacade $facade, SymfonyStyle $i
             throw new \RuntimeException(self::CIRCLE_PULL_REQUEST_ERROR);
         }
 
-        $url = $facade->waitFor($platformId, 'pr-' . $prNum, $sha);
-        $url = rtrim($url, '/');
-        foreach ($files as $file) {
-            try {
-                $this->fileReplace($file, $placeholder, $url, $io);
-            } catch (\RuntimeException $e) {
-                $io->error($e->getMessage());
+        $urls = $facade->waitFor($platformId, 'pr-' . $prNum, $sha);
+        $placeholder_urls = [];
+        foreach ($urls as $index => $url) {
+            $placeholder = ($index === 0) ? "%site-url%" : "%site-url:$index%";
+            $placeholder_urls[$placeholder] = $url;
+        }
+
+        foreach ($placeholder_urls as $placeholder => $url) {
```

Comment:
Update `fileReplace` to take an array of placeholder => url pairs instead. No need to read and write each file times urls.

Reply:
:+1: